src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -50,6 +50,8 @@ internal class HttpApiHandler
         private const string RuntimeStatusParameter = "runtimeStatus";
         private const string PageSizeParameter = "top";
         private const string ReturnInternalServerErrorOnFailure = "returnInternalServerErrorOnFailure";
+        private const string LastOperationTimeFrom = "lastOperationTimeFrom";
+        private const string LastOperationTimeTo = " lastOperationTimeTo";

```

Comment:
It looks like we're missing the `fetchState` query string parameter. This flag determines whether or not to return the entity state. This will be important since state can be potentially large and customers will definitely want a way to avoid paying the overhead of returning the state of all fetched entities (it may be very expensive in terms of CPU and network bandwidth). If not specified, we should assume `false` as the default value for `fetchState`.